gnomad/utils/sparse_mt.py

```diff
@@ -1305,19 +1307,21 @@ def compute_allele_number_per_ref_site(
     :param kwargs: Keyword arguments to pass to `compute_stats_per_ref_site`.
     :return: Table of allele number per reference site.
     """
-    # Determine the genotype field.
     if isinstance(mtds, hl.vds.VariantDataset):
         mt = mtds.variant_data
     else:
         mt = mtds
 
+    # Determine the genotype field.
     en = set(mt.entry)
-    gt_field = (en & {"GT"} or en & {"LGT"}).pop()
+    gt_field = set(mt.entry) & {"GT", "LGT"}
```

Comment:
Following similar logic from above with AD, on the off chance both of these are present, we wouldnt actually know which was used right? .pop() could grab either. I think the old logic makes sense since we would know GT was used if they were both present? I'm not sure it actually matters and its a weird edge case but the code couldnt help you figure out which was used.